Datahub/src/Datahub.Portal/appsettings.json

```diff
@@ -0,0 +1,24 @@
+{
+    "CultureSettings": {
+        "Default": "en-CA",
+        "SupportedCultures": "fr-CA|en-CA"
+    },
+    "LandingBgFolder": "ssc",
+    "LandingBgImgCount": 2,
+    "Title": "Federal Science Datahub",
+    "DataHubModules": "",
+    "DataProjects": {
+        "PowerBI": false,
+        "PublicSharing": false,
+        "WebForms": false,
+        "Databricks": true,
+        "SQLServer": false,
+        "PostgreSQL": true,
+        "Costing": false,
+        "Storage": true
+    },
+    "Achievements": {
+        "Enabled": true,
+        "LocalAchievementsOnly": false
+    }
+}
```

Comment:
We can't include the file, even though there's no sensitive in here (for now)

Reply:
fixed in this commit https://github.com/ssc-sp/datahub-portal/pull/285/commits/727abbb23b6d64a0b06da9a539af723c746a066d